.gitignore

```diff
@@ -1,12 +1,6 @@
-chimera.ini
-hydra.ini
-garage.ini
 *.pyc
-sensorReporter.ini
-sensor_Reporter.yml
-cerberos.ini
-manticore.ini
-iphoned.sh
```

Comment:
Those were still in there? Ha! I don't even have some of those machines any more. I've since moved my configs to a separate repo that I host on my private gitlab so these should be gone. Thanks for fixing this.